webgl.js

```diff
@@ -2794,6 +2831,9 @@ gl.getParameter = function getParameter (pname) {
           case ext.DRAW_BUFFER13_WEBGL:
           case ext.DRAW_BUFFER14_WEBGL:
           case ext.DRAW_BUFFER15_WEBGL:
+            if (ext._buffersState.length === 1 && ext._buffersState[0] === gl.BACK) {
+              return gl.BACK
+            }
```

Comment:
Should the implicit `else` here break or return something specific?

Reply:
It should fall through.  I'll button that up before it is done.  Was in prototyping mode...

![](https://media1.giphy.com/media/XIqCQx02E1U9W/giphy.gif)